Packs/CommonScripts/Scripts/EvaluateContextValue/EvaluateContextValue_test.py

```diff
@@ -0,0 +1,83 @@
+import re
+import json
+import fnmatch
+import pytest
+from typing import Any
+import EvaluateContextValue
+import demistomock as demisto
+
+
+def equals_object(obj1, obj2) -> bool:

```

Comment:
For better type safety and code clarity, please add type hints for the function parameters.